lib/tasks/scheduler.rake

```diff
@@ -43,8 +43,12 @@ end
 desc 'Upload a CSV of a neighborhood'
 task :upload_neighborhood, [:neighborhood_name, :neighborhood_zip_code, :neighborhood_data] => :environment do |t, args|
   starting_count = House.count
-  puts "There are #{starting_count} houses in the database."
-  puts "You have passed in #{args.count} arguments, with these keys: #{args.keys}"
+
+  Rails.logger.info <<~MSG.strip
```

Comment:
Converted to `puts` to `Rails.logger.info` so that logging is silenced in rspec. In order to silence `puts` in rspec, we'd have to mock out `stdout`, and I figured it'd be easier to take advantage of Rails logging instead.

Reply:
Yeah I agree. I didn't really know much about `Rails.logger` when I wrote this code initially so I'm on board.